src/client/mod.rs

```diff
@@ -102,6 +102,18 @@ impl APIClient {
         })
     }
 
+    pub fn request_text(&self, request: http::Request<Vec<u8>>) -> Result<String>
```

Comment:
Could you use `request<String>` as a special case perhaps? I am not sure that would work, but it would avoid this function.

Reply:
https://github.com/clux/kube-rs/blob/a5a10e414518b727df7005bd011f2cfa9f0b31c9/src/client/mod.rs#L88-L90

https://github.com/clux/kube-rs/blob/a5a10e414518b727df7005bd011f2cfa9f0b31c9/src/client/mod.rs#L99-L102

two reasons for why I writting new function:

1. the type of response is bounded by `trait DeserializeOwned` that `type String` not implements
2. the response of the log operation is not in json formatted,

Reply:
Ok, that makes sense.